### PR TITLE
Theseus Update

### PR DESCRIPTION
**Pull request by Alterist**
:cl:
add: Marine Dress Vendors added to Bunks
del: Removed pool, Garden is now in its place
tweak: Tweaked medbay, bridge area, brig, upper engineering
balance:  Slightly weakened SD, removed an autodoc, removed a Cluster OB
fix: Fixed some broken dirs, wrong areas, broken pipes, wiring issues, disposal issues
imageadd: added new red 2x1 doors
/:cl:
incorporates #809 , #752, #725 and #267
gives mirrors dir offests, this might break other maps
There's stacked turfs, but those will have to wait until shuttle rework

### CODE REVIEW

**Comment by Hughgent** [comment]
Looking sweet.

Unless it's a rendering issue. There are no "sleepers" in Medbay lower anymore. (not that I am complaining as I very rarely used them.)

File: code/game/machinery/doors/multi_tile.dm [hunk 1]
```diff
@@ -111,7 +111,12 @@
 	glass = TRUE
 	req_access_txt = "19"
 
-
+/obj/machinery/door/airlock/multi_tile/almayer/secdoor
+	name = "\improper Security Airlock"
+	icon = 'icons/obj/doors/almayer/2x1secdoor.dmi'
+	opacity = FALSE
+	glass = TRUE
+	req_access_txt = "3"
```
Comment (Rohesie) [inline code/game/machinery/doors/multi_tile.dm]: Use regular access, not `txt` one, please.

Comment (Alterist) [reply code/game/machinery/doors/multi_tile.dm]: Changed

**Comment by Rohesie** [comment]
Is there a need to remove dropship sentries?

**Comment by Alterist** [comment]
no, readded

**Pull request merged**